rust-perf/src/lib.rs

```diff
@@ -12,6 +12,16 @@ pub fn run_sha3_256(iterations: usize) {
     }
 }
 
+#[wasm_bindgen]
+pub fn vec_alocation(iterations: usize) {
```

Comment:
Ah, there's a tiny miss-spelling here: `vec_alocation` => `vec_allocation`

Reply:
Sank you!